dist/Views/sidebarview.html

```diff
@@ -114,6 +114,7 @@ <h5>Supported search strings:</h5>
                                 <button ng-show="vm.regionService.selectedRegion.Applications.indexOf('Regulation') != -1" ng-click="vm.checkRegulation()" type="button" class="btn-black fullwidth" ng-disabled="!vm.studyAreaService.regulationCheckComplete" analytics-on analytics-event="Navigation" analytics-category="SideBar" analytics-label="CheckRegulation"><i ng-class="vm.studyAreaService.regulationCheckComplete ? 'fa fa-flash' : 'fa fa-refresh fa-spin'"></i>&nbsp;&nbsp;Check for upstream regulation</button>
                                 <button ng-show="vm.regionService.selectedRegion.Applications.indexOf('StormRunoff') != -1" ng-click="vm.OpenStormRunoff()" type="button" class="btn-black fullwidth"><i class="fa fa-steam" analytics-on analytics-event="Navigation" analytics-category="SideBar" analytics-label="StormRunoff"></i>&nbsp;&nbsp;Open Storm Runoff Models</button>
                                 <button ng-show="vm.regionService.selectedRegion.Applications.indexOf('SCStormRunoff') != -1" ng-click="vm.OpenSCStormRunoff()" type="button" class="btn-black fullwidth"><i class="fa fa-steam" analytics-on analytics-event="Navigation" analytics-category="SideBar" analytics-label="SCStormRunoff"></i>&nbsp;&nbsp;Open Storm Runoff Models</button>
+                                <button ng-show="vm.regionService.selectedRegion.Applications.indexOf('HydrologicFeatures') != -1" ng-click="vm.OpenSelectHydrologicFeatures()" type="button" class="btn-black fullwidth"><i class="fa fa-steam" analytics-on analytics-event="Navigation" analytics-category="SideBar" analytics-label="HydrologicFeatures"></i>&nbsp;&nbsp;Select Hydrologic Features</button>

```

Comment:
After you finish making changes I think you should try to rebuild the application so we don't get this old dist file pulled back into the branch. The instructions are in the readme, https://github.com/USGS-WiM/StreamStats#building-and-testing, you should just have to run `npm run-script build`. 

I guess I am not sure if you ever got approved for a pr account but I think that command might need to be run from an admin command prompt. If you try and it doesn't work Andrea or I can rebuild this branch before we merge.